src/scripts/activate-user.ts

```diff
@@ -88,6 +103,33 @@ const validateUserStatusPriorToActivation = (
   }
 };
 
+/**
+ * Plans the commands to execute to activate the user.
+ * @param service - the service to use to activate the user.
+ * @param userId - the id of the user to activate.
+ * @param sendEmail - if true, will send activation email to the user.
+ * @returns a TaskEither that resolves to an array of commands to execute to activate the user.
+ */
+const planCommands = (
+  service: UserService,
+  userId: string,
+  sendEmail: boolean
+): TE.TaskEither<string, readonly Command[]> =>
```

Comment:
**Issue:** sorry if I have confused things. In this case, you only need to return a single `Command`. You don't have to generalise this to cater for situations that we know won't exist in this context.